src/main/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/service/RiskService.kt

```diff
@@ -89,7 +89,9 @@ internal class RiskService(
       return AssessmentInfo(error = extractErrorCode(ex, "risk assessments", crn))
     }
     val latestCompleteAssessment = getLatestCompleteAssessment(assessmentsResponse)
-    val mainActiveCustodialOffenceFromLatestCompleteAssessment = getMainActiveCustodialOffenceFromLatestCompleteAssessment(activeConvictionsFromDelius, latestCompleteAssessment)
+    val mainActiveCustodialOffenceFromLatestCompleteAssessment = if (hideOffenceDetailsWhenNoMatch) {
```

Comment:
why is there different rules between when it comes from the RecommendationService and when it comes from the CaseSummaryOverviewService? Can't see an AC in MRD-700 to cover that so just wanted to make sure this is intended?

Reply:
It's not written in the AC. Let me dig out some Slack convos I've had with Andre and update the ticket with some references.